.travis.yml

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+language: java
+
+# blacklist
+branches:
+  except:
+    - asf-site
+    - gh-pages
+
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode8
+
+before_install:
+  - echo "MAVEN_OPTS='-Xmx3072m -XX:MaxPermSize=512m'" > ~/.mavenrc
+
+script:
+  - travis_retry mvn --batch-mode clean apache-rat:check
+  - travis_wait 60 mvn --batch-mode clean package findbugs:check
```

Comment:
running apache rat + findbugs in a single run will speed up the process
for findbugs the "compile" goal would be enough, for rat I don't know, 
we can skip the assembly plugin so we can make the check faster